packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/examples.md

```diff
@@ -0,0 +1,14 @@
+### Examples
+
+The following web application examples show you the Authorization Code flow, as it would be implemented by a web app that needs to authenticate the end user and then create a local session for that user. These projects use popular web frameworks to handle the heavy lifting. Each project can be cloned and ran locally.
```

Comment:
```suggestion
The following web application examples show you the Authorization Code flow as it would be implemented by a web app that needs to authenticate the end user and then create a local session for that user. These projects use popular web frameworks to handle the heavy lifting. Each project can be cloned and ran locally.
```